docs/console/guides/blueprint/blueprint-configuration.md

```diff
@@ -221,6 +259,106 @@ Resource successfully created in project 'ash-devnet'!
++=============+=============+========+==================+=========+
+| ewoq-key    | e9b0...5e35 | Wallet | 2024-01-30T17:25 | 0       |
++-------------+-------------+--------+------------------+---------+
+
+Updating entities...
+Updating project: ash-devnet
+Project updated successfully!
++--------------+-------------+---------+-----------------+--------------------+------------------+
+| Project name | Project ID  | Network | Cloud regions   | Resources          | Created at       |
++==============+=============+=========+=================+====================+==================+
+| ash-devnet   | 3443...5326 | Local   |  aws/us-east-1  |  avalancheNode: 5  | 2024-01-30T13:21 |
++--------------+-------------+---------+-----------------+--------------------+------------------+
+
+Adding resource: ash-devnet:ash-subnet
+Resource successfully created in project 'ash-devnet'!
++---------------+-------------+-----------------+---------------+-------+------------------+---------+-------------------+
+| Resource name | Resource ID | Type            | Cloud region  | Size  | Created at       | Status  | Resource specific |
++===============+=============+=================+===============+=======+==================+=========+===================+
+| ash-subnet    | 396d...6cc8 | AvalancheSubnet | aws/us-east-1 | Small | 2024-01-30T17:26 | Pending |  ID         |     |
+|               |             |                 |               |       |                  |         |  Validators | 0   |
++---------------+-------------+-----------------+---------------+-------+------------------+---------+-------------------+
+```
+
+## Subnet ID
+
+After a few minutes, we can get the Subnet ID of our Subnet with the `console resource info` command:
+
+```bash title="Command"
+ash console resource info ash-subnet
+```
+
+```bash title="Output"
+Resource 'ash-subnet' of project 'ash-devnet':
++---------------+-------------+-----------------+---------------+-------+------------------+---------+-------------------------------------------------------------------+
+| Resource name | Resource ID | Type            | Cloud region  | Size  | Created at       | Status  | Resource specific                                                 |
++===============+=============+=================+===============+=======+==================+=========+===================================================================+
+| ash-subnet    | 396d...6cc8 | AvalancheSubnet | aws/us-east-1 | Small | 2024-01-30T17:26 | Running |  ID         | 2rArnpKkN9fKch9Q6ana3awAfVmM4HfLoNYrZZDkSumApEi8yZ  |
+|               |             |                 |               |       |                  |         |  Validators | 5                                                   |
++---------------+-------------+-----------------+---------------+-------+------------------+---------+-------------------------------------------------------------------+
+```
```

Comment:
## Subnet information

After a few minutes, we can get the Subnet ID, control keys, blockchains, and validator nodes information from the `subnetStatus` field returned by the `console resource info` command:

```bash title="Command"
ash console resource info my-subnet-evm --json | jq '.subnetStatus'
```

:::tip
We use the `--json` flag to get the extended information about the Subnet. All those information cannot be displayed in a table format.
:::

```python title="Output"
{
  "blockchains": [
    {
      "id": "RfX8YRUHePxFENBZFwaNt1tP6CR6RQ4hJJ5AqYv49LRoSKniK",
      "name": "AshLocalEVM",
      "vm_id": "srEXiWaHuhNyGwPUi444Tu47ZEDwxTWrbQiuD7FmgSAQ6X7Dy",
      "vm_type": "SubnetEVM"
    }
  ],
  "controlKeys": [
    "P-local18jma8ppw3nhx5r4ap8clazz0dps7rv5u00z96u"
  ],
  "id": "29uVeLPJB1eQJkzRemU8g8wZDw5uJRqpab5U2mX9euieVwiEbL",
  "pendingValidators": [],
  "subnetType": "Permissioned",
  "threshold": 1,
  "validators": [
    {
      "connected": true,
      "end_time": 1707486609,
      "node_id": "NodeID-MFrZFVCXPv5iCn6M9K6XduxGTYp891xXZ",
      "stake_amount": 100,
      "start_time": 1706881809,
      "tx_id": "DhEgYijBur6QDUDgUMpmDAyF5XGyq6JwG919B8bkryDLijvua",
      "uptime": 100,
      "weight": 100
    },
    # ...
  ]
}
```

Reply:
Cannot do a suggestion on a README with code bocks

Reply:
Done in https://github.com/AshAvalanche/ash-docs/pull/79/commits/6df71bc71fe5f55f263dfd532ef4b6b885d72d2b